static/js/components/Source.jsx

```diff
@@ -120,6 +124,15 @@ const Source = ({ route }) => {
         />
         <br />
         <br />
+        <Responsive
```

Comment:
I personally prefer `@material-ui/core/Accordion` and would replace all occurrences of `Responsive` in SP with it, but this needs to be run by the Jedi Masters. I'll introduce those in my next PR, we'll say what they have to say...

Reply:
Yes I also thought about that, but that felt like another PR as that involves the rest of the page so I kept it consistent for now.